packages/javascript/bh-shared-ui/src/components/GroupManagementContent/GroupManagementContent.tsx

```diff
@@ -80,6 +83,24 @@ const GroupManagementContent: FC<{
         return selectedAssetGroup?.name || 'Select a Group';
     };
 
+    const handleFilterChange = (
+        key: keyof Pick<AssetGroupMemberParams, 'primary_kind' | 'custom_member'>,
+        value: string
+    ) => {
+        // Custom Member filter displays custom members, or all members.
+        // If we want to also display only non customer members, change this:
+        if (key === 'custom_member' && value.toLowerCase().includes('false')) {
+            setFilterParams(({ custom_member, ...removedCustomMemberFilter }) => removedCustomMemberFilter);
+            return;
```

Comment:
Is this backwards?  The logic escapes me a bit, but should we be setting filter params to `custom_member` without the `removedCustomMemberFilter`?

Reply:
Great question! In the case that the value of the checkbox is `'false'` (not just falsey 🙄 ) then we want to remove the filter param completely because the two states with this filter are: 1. only show custom members, or 2. show all members. If we set the `custom_member` param to 'false', then it will return all non-custom members. Which is not the intended use case according to product.

So here we are deconstructing the existing `filterParams` object pulling `customer_member` out and renaming the rest of the params to `removedCustomMemberFilter`. That way we can return `removedCustomMemberFilter` without the customer_member param! Now that im looking at it again, its not a very good name :/ 

Ill break this line up into a couple lines to make is a little more explicit as to whats going on!